static/js/services/all-contacts.js

```diff
@@ -1,7 +1,7 @@
 var _ = require('underscore'),
     async = require('async');
 
-angular.module('inboxServices').factory('Facility',
+angular.module('inboxServices').factory('AllContacts',
```

Comment:
Maybe just `Contacts`? Seeing you can filter by type `AllContacts` doesn't seem right?